bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -70,6 +76,9 @@ public OpenWebNetEnergyHandler(Thing thing) {
     public void initialize() {
         super.initialize();
 
+        Object refreshPeriodConfig = getConfig().get(OpenWebNetBindingConstants.CONFIG_PROPERTY_REFRESH_PERIOD);
+        energyRefreshPeriod = Integer.parseInt(refreshPeriodConfig.toString());
```

Comment:
By security, please set a default value in case the parsing is failing:
```
try {
    Object refreshPeriodConfig = getConfig().get(OpenWebNetBindingConstants.CONFIG_PROPERTY_REFRESH_PERIOD);
    energyRefreshPeriod = Integer.parseInt(refreshPeriodConfig.toString());
} catch (NumberFormatException e) {
    energyRefreshPeriod = 30;
}
```

Reply:
ok I also added a log message:
logger.warn("NumberFormatException on parsing OpenWebNetEnergyHandler configuration: {}",  e.getMessage());

Reply:
logger.debug("NumberFormatException caught while parsing OpenWebNetEnergyHandler configuration: {}", e.getMessage());

Reply:
Normally it will never happen.
DEBUG level is better, yes.